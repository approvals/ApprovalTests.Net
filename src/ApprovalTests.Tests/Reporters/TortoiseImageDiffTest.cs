using ApprovalTests.Reporters.Windows;
using NUnit.Framework;

namespace ApprovalTests.Tests.Reporters
{
    [TestFixture]
    public class TortoiseImageDiffTest
    {
        [Test]
        public void TestIsImage()
        {
            if (!TortoiseTextDiffReporter.INSTANCE.IsWorkingInThisEnvironment("a.txt"))
            {
                return;
            }

            var files = new[]
            {
                "image.png", "image.gif", "image.jpg", "image.jpeg", "image.tif", "image.tiff",
                "movie.avi", "text.txt", "excel.xls"
            };
            Approvals.VerifyAll(
                files,
                f => $"{f}  => {TortoiseImageDiffReporter.INSTANCE.IsWorkingInThisEnvironment(f)}");
        }
    }
}