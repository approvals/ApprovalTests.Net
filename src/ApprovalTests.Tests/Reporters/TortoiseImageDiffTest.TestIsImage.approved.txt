image.png  => True
image.gif  => True
image.jpg  => True
image.jpeg  => True
image.tif  => True
image.tiff  => True
movie.avi  => False
text.txt  => False
excel.xls  => False
